source/blog/2020-06-16-rspec-branching-strategy.md

```diff
@@ -0,0 +1,56 @@
+---
+title: RSpec Branching Strategy
+author: Jon Rowe
+---
+
+# The short version.
```

Comment:
Maybe remove the trailing periods from titles?

Reply:
I agree looks better